### PR TITLE
New Bootstrap Cheat Sheet

### PR DESCRIPTION
**Pull request by ethanchewy**
## Demo

Demo GIF

Demo link: https://drive.google.com/file/d/0B_0VuTZLjvTQUWZBYTNYT1hkckk/view?usp=sharing

**What does your Instant Answer do?**
It gives a quick description of core CSS classes in the Boostrap frameworks. 

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It is much better than organic links IMO because I aggregated this information from StackOverflow, w3schools, and the Bootstrap documentation. I thought that for a quick reference, the bootstrap documentation can be confusing so I sought to make this easier by explaining it a little bit more specifically.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Mainly http://getbootstrap.com/ as a guide but I also used http://www.w3schools.com/bootstrap, , http://stackoverflow.com/

**Why did you choose this data source?**
I chose these data sources because they are well known and accurate (for stackoverflow, I made sure that the information I pulled was from a reputable commenter).

**Are there any other alternative (better) data sources?**
 http://www.w3schools.com/bootstrap, , http://stackoverflow.com/

**What are some example queries that trigger this Instant Answer?**
"twitter front end framework", "twitter blueprint"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Front-End developers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Unfortunately no. I didn't get the go ahead for my post in the forums yet unfortunately but I really wanted to make this :) (https://duck.co/ideas/idea/6655/bootstrap-cheatsheet)

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**
Not really

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 10
        - [X] Google Chrome
        - [X] Firefox
        - [X] Opera
        - [X] IE 11
        - [] Microsoft edge
        
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Linux
        - [] Google Chrome
        - [] Firefox
        
    - Mac OSX
        - [X] Google Chrome
        - [X] Firefox
        - [X] Opera
        - [X] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [X] Google Chrome
        - [X] Opera

```

-----
IA Page: https://duck.co/ia/view/bootstrap_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/bootstrap.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
+{
+    "id": "bootstrap_cheat_sheet",
+    "name": "Bootstrap",
+    "description": "Explanation of Bootstrap Classes",
+    "metadata": {
+        "sourceName": "Bootstrap",
+        "sourceUrl": "http://getbootstrap.com/"
+    },
+    "aliases": [
+        "twitter front end framework", "twitter blueprint"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/bootstrap.json]: I would add `twitter bootstrap`

**Comment by moollaza** [comment]
@ethanchewy BTW, there's a slightly different Pull Request Template for Cheat Sheets in the repo. It doesn't have as many questions or the checklist. You'll probably want to use that one in the future ;)

**Comment by ethanchewy** [comment]
@moollaza Good to know. Will do!

**Comment by ethanchewy** [comment]
Oops, accidentally closed the repo, sorry about that!

**Comment by moollaza** [comment]
@ethanchewy this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@ethanchewy congrats! It's live: https://duckduckgo.com/?q=Bootstrap%20cheat%20sheet&ia=cheatsheet (if not live yet, it will be online very soon...)

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**